src/components/Header/Header.test.js

```diff
@@ -6,7 +6,10 @@ import { HeaderUnwrapped } from './Header';
 describe('Header', () => {
   it('should render Header correctly', () => {
     const onSetFilterSpy = jest.fn();
-    const wrapper = shallow(<HeaderUnwrapped onSetFilter={onSetFilterSpy} />);
+    const onStartLogout = jest.fn();
+    const wrapper = shallow(
+      <HeaderUnwrapped onSetFilter={onSetFilterSpy} onStartLogout={onStartLogout} />,
```

Comment:
why there is no assertion added to verify if this function has been invoked properly when something is clicked?

Reply:
this is one of those tricky-son-of-a-bitch test that I can't handle. I will get back to it in future PRs related specifically to tests.